README.md

```diff
@@ -406,6 +406,70 @@ Loans include information from the item, including locations, holdingsRecordId a
 
 Requests include information from the item, including holdingsRecordId and instanceId.
 
+#### Hold shelf clearance report CSV
+
+To create first pass hold expiration report that can be used by staff to clear expired and cancelled holds from the shelf and put them back into circulation.
```

Comment:
Does "first pass" have some meaning that I am not aware of? I thought we were using "first pass" to indicate that this was a stop gap to a better implementation in the future. Do we want to advertise that fact? I see "first pass" in other places here, like the permission. If it has some sort of library meaning, then ignore this. 😄

Reply:
To me, this refers to this being an intermediary, workaround solution. I don’t think this phrase should appear in the domain or API

Reply:
Fixed